src/admin/auth/index.js

```diff
@@ -11,7 +11,7 @@ exports.plugin = {
     // server.auth.default('simple')
     server.auth.strategy('jwt-strategy', 'hapi-now-auth', {
       verifyJWT: true,
-      keychain: [Config.ADMIN_SECRET],
```

Comment:
Why has the config reference been removed and hardcoded to 'secret'?

Reply:
reverted